app.ts

```diff
@@ -29,10 +31,15 @@ app.use(morgan('combined', {stream: {
   write: function(message) { log.info('HTTP: ' + message); }
 }}));
 
+var dirname = '/Users/plipsanen/personal/nappikauppa2'; // TODO
```

Comment:
hmm `__dirname` seems to work now, I'm pretty sure it was broken in typescript just a moment ago...